tests/test_pipenv.py

```diff
@@ -1167,11 +1168,13 @@ def test_environment_variable_value_does_not_change_hash(self, pypi):
     def test_scripts_basic(self):
         with PipenvInstance(chdir=True) as p:
```

Comment:
Pretty much this entire test is invalid for windows -- you can't escape double quotes on windows, period. So There is no way to make this test pass on windows.

Reply:
yeah I think you should just skip the second half or split it in two - not super familiar with windows